### DISCUSSION TITLE
LG devices unlock capabilities

### DISCUSSION THREAD

**Discussion by SirRGB** [openandroidinstaller-dev/openandroidinstaller#445] [Q&A] [answered]
Not sure why LG devices are listed as unsupported in the unlocking section
https://github.com/openandroidinstaller-dev/openandroidinstaller/blob/main/docs/unlocking_the_bootloader.md?plain=1#L26

The lineage wiki lists them as officially supported with standard fastboot unlock procedure. Did I miss anything?
https://wiki.lineageos.org/devices/#lg

Edit: just read the french source (translated), but I still cant really understand why unlocking is deemed to be impossible.

**Comment by tsterbak** [comment, marked as answer]
The reason for this is, that LG stopped the official support for bootloader unlocking end of 2021: https://www.xda-developers.com/lg-bootloader-unlocking-shut-down-december-31/

There might be unofficial ways to unlock certain devices (and some people already have unlocked devices), but the procedure is usually more difficult than flashing LOS. So I considered LG devices out of the target group.

LineageOS also shows this on every LG device page:
> Important: An official method of unlocking the bootloader is not available, but there may be some unofficial methods available. Unofficial methods are not recommended or supported by LineageOS, so use them at your own risk! An unlocked bootloader is required to install LineageOS.

But in general, if you want to add support for an LG device, I'm happy to include it.
